rotkehlchen/chain/ethereum/utils.py

```diff
@@ -223,13 +222,3 @@ def try_download_ens_avatar(
     avatars_dir.mkdir(exist_ok=True)  # Ensure that the avatars directory exists
     with open(avatars_dir / f'{ens_name}.png', 'wb') as f:
         f.write(avatar)
-
-
-def has_tracked_addresses(
-        blockchain: SupportedBlockchain,
-        database: 'DBHandler',
-) -> bool:
-    with database.conn.read_ctx() as cursor:
-        return database.get_blockchain_accounts(cursor).get(
-            blockchain=blockchain,
-        ) != 0
```

Comment:
This function was broken and never worked. Always returned true.

Blockchain accounts is an object of `BlockchainAccounts`. Will never be 0.